### DISCUSSION TITLE
Updating stale links on jdotxt.org

### DISCUSSION THREAD

**Discussion by t7ko** [todotxt/todo.txt-cli#374] [Q&A] [answered]
Hi All,

I am now maintaining one of the apps linked to on http://todotxt.org.  Old maintainer does not seem to be interested in maintaining it further.  I'm searching for a way to update the link on http://todotxt.org.  Can anyone please help me here?  Any hints highly appreciated, thanks in advance.

Details:  The app is jdotxt.  Original link: https://github.com/chms/jdotxt .  Last updated 3 years ago.  My fork: https://github.com/t7ko/jdotxt .  Original link does not even allow to download the app any more -- downloads were hosted on author's personal home page, which is no longer available.  In my fork, I'm posting releases to GitHub -- supposed to be much more "permanent".

**Comment by karbassi** [comment, marked as answer]
Hello @t7ko!

You can submit a PR to https://github.com/todotxt/todotxt.github.io
